src/pyrobot/core.py

```diff
@@ -74,6 +74,7 @@ def __init__(self,
         for f in os.listdir(cfg_path):
             if f.endswith('_config.py'):
                 robot_pool.append(f[:-len('_config.py')])
+        print(robot_pool)
```

Comment:
I guess this is for debugging and should be removed?

Reply:
Thank you so much. I removed that.